go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/apache/dubbo-go
 
+go 1.14
```

Comment:
pld do not add go version in the go.mod.

Reply:
agree